huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -16,7 +16,9 @@ import (
 	"github.com/chnsz/golangsdk"
 	"github.com/chnsz/golangsdk/openstack/ecs/v1/block_devices"
 	"github.com/chnsz/golangsdk/openstack/ecs/v1/jobs"
+	evsjobs "github.com/chnsz/golangsdk/openstack/evs/v1/jobs"
```

Comment:
it's better to add the alias for "ecs/v1/jobs" package

```suggestion
	ecsjobs "github.com/chnsz/golangsdk/openstack/ecs/v1/jobs"
	"github.com/chnsz/golangsdk/openstack/evs/v1/jobs"
```

Reply:
Done